examples/client.rs

```diff
@@ -30,22 +30,23 @@ fn main() {
         return;
     }
 
-    let mut core = tokio_core::reactor::Core::new().unwrap();
-    let handle = core.handle();
-    let client = Client::new(&handle);
+    tokio::run(lazy(move || {
```

Comment:
I'm curious what the `lazy` usage is for. Is that because calling all this code outside of `tokio::run` will panic?

Reply:
Yeah, most constructors, including `Client::default` now need a thread-local handle. When we change the API I think we should defer all I/O (like port binding) until the future is polled, but for now that's the compromise I made.